samples/005-PreDebrief/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "SuspensionMng",
+  "version": "1.1.0",
+  "description": "Template for Oracle Field Service plugins",
+  "main": "index.js",
+  "scripts": {
+    "test": "jest"
+  },
+  "keywords": [
+    "ofs",
+    "ofsc"
+  ],
+  "author": "miguel.gall@oracle.com",
+  "license": "ISC",
+  "devDependencies": {
+    "@types/jest": "^29.4.0",
+    "@types/node-sass": "^4.11.3",
+    "css-loader": "^6.7.3",
+    "node-sass": "^8.0.0",
+    "sass": "^1.57.1",
+    "sass-loader": "^13.2.0",
+    "style-loader": "^3.3.1",
+    "ts-jest": "^29.0.5",
+    "ts-loader": "^9.4.2",
+    "typescript": "^4.9.4",
+    "webpack": "^5.75.0",
+    "webpack-cli": "^5.0.1"
+  },
+  "dependencies": {
+    "@ofs-users/plugin": "^0.11.1",
```

Comment:
Please update the dependencias, those are part of an old version of the template:

    "@ofs-users/plugin": "^1.0.0",
    "@ofs-users/proxy": "^1.4.0",